src/components/questions/sectionizer.cjsx

```diff
@@ -0,0 +1,55 @@
+React = require 'react'
+cn = require 'classnames'
+ScrollTo = require '../scroll-to-mixin'
+
+Sectionizer = React.createClass
+
+  mixins: [ScrollTo]
+
+  propTypes:
+    chapter_sections: React.PropTypes.array.isRequired
+    onScreenElements: React.PropTypes.array.isRequired
+
+  getInitialState: ->
+    {scrollingTo: _.first(@props.chapter_sections)}
+
+  # the below properties are read by the ScrollTo mixin
+  scrollingTargetDOM: -> window.document
+  getScrollTopOffset: -> 140 # 70px high control bar and 50px spacing for label
+
+  componentDidMount: ->
+    @scrollToSelector(".questions-list")
+
+  scrollToSection: (section) ->
+    @scrollToSelector("[data-section='#{section}']")
+
+  scrollIndex: ->
+    @props.chapter_sections.indexOf(@props.chapter_sections)
+
+  goBack: ->
+    index = @scrollIndex()
+    @scrollToSection(sections[ if index > 1 then index - 1 else 0])
```

Comment:
Great catch, thanks!  That was missed when I extracted the "Sectionizer" component out from where it used to be inline on the controls.